code/modules/client/preferences_savefile.dm

```diff
@@ -73,6 +73,19 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		if(clientfps == 0)
 			clientfps = -1
 
+	if (current_version < 38)
+		var/found_block_movement = FALSE
+
+		look_through_bindings:
+			for (var/list/key in key_bindings)
+				for (var/bind in key)
+					if (bind == "block_movement")
+						found_block_movement = TRUE
+						break look_through_bindings
```

Comment:
```suggestion
		for (var/list/key in key_bindings)
			for (var/bind in key)
				if (bind == "block_movement")
					found_block_movement = TRUE
					break
			if(found_block_movement)
				break
```
goto very evil

Reply:
I was told named breaks are okay, but I can live with this.

Reply:
I suppose we don't have a hard ruling on this. I dislike it immensely. Unless you are running very hot code, there's absolutely no need for this, you can always replicate the behavior with a control var.